src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -55,11 +56,11 @@ class TextGenerationInput(BaseModel):
         "the logits for the input text sequence and the "
         "generated text sequence. ",
     )
-    session_id: Optional[str] = Field(
+    session_ids: Union[None, List[str], str] = Field(
```

Comment:
let's talk more on this - if we have a batched input does this mean each batch item should have its own session / kv cache stored?

Reply:
yes, at least for now. happy to discuss the UI if this is not the case.